packages/suite/src/components/wallet/WalletLayout/AccountsMenu/AccountsList.tsx

```diff
@@ -37,7 +39,13 @@ export const AccountsList = ({ onItemClick }: AccountListProps) => {
     const list = sortByCoin(accounts.filter(a => a.deviceState === device.state).concat(failed));
     const filteredAccounts =
         searchString || coinFilter
-            ? list.filter(a => accountSearchFn(a, searchString, coinFilter, accountLabels[a.key]))
+            ? list.filter(account => {
+                  const { key, accountType, symbol, index } = account;
+                  const accountLabel =
+                      accountLabels[key] || getDefaultAccountLabel({ accountType, symbol, index });
```

Comment:
Nit: this is imho more semantic - although it's longer, it makes it clear we're not checking for truthiness, but if the value exists on the record at all. Though this code is also fine :ok_hand: 
```suggestion
                      accountLabels.hasOwnProperty(key) ? accountLabels[key] : getDefaultAccountLabel({ accountType, symbol, index });
```

Reply:
I really like this solution. 🤩